internal/archiver/archiver_test.go

```diff
@@ -2061,8 +2061,8 @@ func TestMetadataChanged(t *testing.T) {
 
 	// set some values so we can then compare the nodes
 	want.Content = node2.Content
+	want.ExtendedAttributes = node2.ExtendedAttributes
```

Comment:
This test failed on my machine due to selinux:

```txt
--- FAIL: TestMetadataChanged (0.00s)
    testing.go:30: using low-security KDF parameters for test
    config.go:65: disabling check of the chunker polynomial
    archiver_test.go:2043: chdir to /tmp/restic-test-637943804
    archiver_test.go:2069: metadata does not match:
        {*restic.Node}.ExtendedAttributes:
                -: []restic.ExtendedAttribute(nil)
                +: []restic.ExtendedAttribute{{Name: "security.selinux", Value: []uint8{0x75, 0x6e, ..., 0x30, 0x00}}}
    panic.go:406: chdir back to /home/pschultz/src/restic/internal/archiver
FAIL
FAIL    github.com/restic/restic/internal/archiver      2.598s
```

I'm not sure why this test expected the ExtendedAttributes to be nil after snapshotting. [According to the manual ExtendedAttributes are preserved](https://github.com/restic/restic/blob/master/doc/manual_rest.rst#metadata-handling).

> Restic saves and restores most default attributes, including extended attributes like ACLs.

Reply:
My guess would be that it made the test work. If the node has no ExtendedAttribute, then the node create from NodeFromFileInfo has a value of `[]restic.ExtendedAttribute()`. The node loaded from the repository has `[]restic.ExtendedAttribute(nil)`. These two values differ, causing the test to fail.

The Lstat wrapper can't control the ExtendedAttributes, therefore your change seems to be the only easy way to fix the problem.

Reply:
Leave it in for now, I don't see an easier way.

Reply:
This problem has been fixed on master. Could you please rebase onto the current master and remove the test fix?

Reply:
Done.